noetic/distribution.yaml

```diff
@@ -884,6 +884,21 @@ repositories:
       url: https://github.com/swri-robotics/gps_umd.git
       version: master
     status: maintained
+  graph_msgs:
+    doc:
+      type: git
+      url: https://github.com/PickNikRobotics/graph_msgs.git
```

Comment:
The previous versions of this package used the URL as https://github.com/davetcoleman/graph_msgs.git .  It's OK to move them, but I would like to see:

a)  That @davetcoleman approves of this, and
b)  Updating Melodic and Kinetic to point to this repository

Reply:
I approve. PickNik is the company we founded :-)

Reply:
Thanks, appreciated.